literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml

```diff
@@ -0,0 +1,76 @@
+# This file is part of echemdb (https://github.com/echemdb/website)
+# Details on the meta data structure can be found on https://github.com/echemdb/metadata-schema
+
+experimental:
+  instrumentation:
+  - type: potentiostat
+
+system:
+  type: electrochemical
+  electrolyte:
+    type: aq
+    components:
+      - name: water
+        sum formula: H2O
+        type: solvent
+        proportion:
+          value: 100
+          unit: volume percent
+        source:
+          refinement: Milli-Q SP-TOC
+          quality: ultrapure water
+        purity:
+          resistance:
+            value: 18.2
+            unit: MOhm cm
+      - name: sulfuric acid
+        sum formula: H2SO4
+        type: acid
+        source:
+          supplier: Cica-Merck
+          grade: ultrapur
+          LOT: null
+        concentration:
+          value: 1.0
+          unit: mol L⁻¹
+    pH:
+      estimated value (CurTiPot): 0.1

```

Comment:
I think this should always be `value`.
Additional information could be stored in a key called `comment'.